src/pybind/mgr/dashboard/frontend/src/app/shared/components/orchestrator-doc-panel/orchestrator-doc-panel.component.html

```diff
@@ -1,5 +1,10 @@
-<cd-alert-panel type="info"
-                i18n>Orchestrator is not available. Please consult the
+<cd-alert-panel *ngIf="missingFeatures; else elseBlock"
+                type="info"
+                i18n>The feature is not supported in the current orchestrator.</cd-alert-panel>
```

Comment:
If it is a name in this context it should be upper case.
```suggestion
                i18n>The feature is not supported in the current Orchestrator.</cd-alert-panel>
```

Reply:
Fixed!